django_mesh/models.py

```diff
@@ -191,4 +194,48 @@ class MockYoutube(models.Model):
     you = models.CharField(max_length=140, unique=True)
     yoh = models.CharField(max_length=140, unique=True)
     yot = models.TextField(default='')
-    yoj = models.TextField(default='')
\ No newline at end of file
+    yoj = models.TextField(default='')
+
+class Media(_Abstract):
+
+    MEDIA_TYPE = Choices(
```

Comment:
Hey Sean, Everything else (Post, Channel, Tag, etc) has been singular. Should we keep it singular to keep it consistant?